src/mscorlib/src/System/String.Manipulation.cs

```diff
@@ -1001,9 +1001,7 @@ public String[] Split(char[] separator, int count, StringSplitOptions options)
         [System.Security.SecuritySafeCritical]
         private unsafe String[] SplitInternal(char separator, int count, StringSplitOptions options)
```

Comment:
It may not be needed once we have `Span<T>` and C# compiler support to allow taking address of local as `Span<T>` ,